build.sh

```diff
@@ -168,16 +129,13 @@ main() {
     #  Export some stuff for maven.
     export MVN MVN_OPTS MVN_GOAL
 
-    echo "$MVN $MVN_OPTIONS $MVN_GOAL $ADDIT_PARAMS"
-
-    # workaround in case 'mvn -f' is not supported
-    if [ "$PRESERVE_WORKING_DIR" = "true" ]; then cd "$ORIG_WORKING_DIR"; fi
+    echo "${BASH_INTERPRETER} $MVN $MVN_OPTIONS $MVN_GOAL $ADDIT_PARAMS"
```

Comment:
I am not sure who set PRESERVE_WORKING_DIR before, but now it won't allow the cd into ORIG_WORKING_DIR as part of the script execution?

Reply:
I'm not sure why `PRESERVE_WORKING_DIR` was introduced. Also, I don't think we use this env var anywhere. Moreover, `mvnw` can only be invoked from one place, i.e. the root of Narayana's code base. Sorry if I missed something obvious.